tests/jdom/fastforwardtester.ts

```diff
@@ -84,10 +85,12 @@ export class FastForwardTester
     // start() and stop() are made available, but recommend using withTestBus() instead
     public start() {
         this.bus.start()
+        this.scheduler.start() // run in background
     }
 
     public stop() {
         this.bus.stop()
+        this.scheduler.stop()
```

Comment:
same comment

Reply:
Yeah, so the issue is that there isn't a `start`/`stop` interface for the base `Scheduler`, and `stop` doesn't really make sense for the wall clock scheduler.

Potential options include:
- keep as-is - which might be the least worst option, since FastForwardScheduler is tightly integrated with the FastForwardTester anyways
- have the FastForwardScheduler automatically start on construction, but keep the explicit `stop` so it can be removed from the event queue
- add a start/stop interface on the schedulers, and have the WallClockScheduler no-op when asked to `stop` - which seems like a poor API design
- actually have the WallClockScheduler be `stop`pable - which is clean, but doesn't seem to be useful
- add an intermediate class of StartStopScheduler, which FastForwardScheduler would extend (but WallClockScheduler would not), and which the bus is aware of and can forward `start` and `stop` calls